### PR TITLE
Append CSN info in RemoteXact request

### PR DESCRIPTION
**Pull request by poojanilangekar**

### CODE REVIEW

File: contrib/remotexact/rwset.c [hunk 1]
```diff
@@ -232,8 +237,8 @@ RWSetToString(RWSet *rwset)
 	/* Header */
 	header = &rwset->header;
 	appendStringInfoString(&s, "{\n\"header\": ");
-	appendStringInfo(&s, "{ \"dbid\": %d, \"xid\": %d, \"region_set\": %ld }",
-					 header->dbid, header->xid, header->region_set);
+	appendStringInfo(&s, "{ \"dbid\": %d, \"xid\": %d, \"csn\": %d, \"region_set\": %ld }",
```
Comment (ctring) [inline contrib/remotexact/rwset.c:240]: Use `%ld`. Change the per-page csn below, too.

File: contrib/remotexact/remotexact.c [hunk 2]
```diff
@@ -261,6 +261,7 @@ rx_send_rwset_and_wait(void)
 	header = &rwset_collection_buffer->header;
 	pq_sendint32(&buf, header->dbid);
 	pq_sendint32(&buf, header->xid);
+	pq_sendint64(&buf, header->csn);
```
Comment (ctring) [inline contrib/remotexact/remotexact.c:264]: Could you also update the line that sends csn in `rx_collect_page` to use `pq_sendint64`?

File: contrib/remotexact/rwset.h [hunk 3]
```diff
@@ -21,7 +21,8 @@ typedef struct RWSetHeader
 {
 	Oid			dbid;
 	TransactionId xid;
-	uint64		region_set;
+	uint64 		csn;
+	uint64 region_set;
```
Comment (ctring) [inline contrib/remotexact/rwset.h:25]: nit: formatting

**Review by ctring — APPROVED** [review]
Looks good to me.

File: contrib/remotexact/rwset.c (hunk 1 above)
Comment (poojanilangekar) [reply contrib/remotexact/rwset.c:240]: Done.

File: contrib/remotexact/remotexact.c (hunk 2 above)
Comment (poojanilangekar) [reply contrib/remotexact/remotexact.c:264]: Done.

File: contrib/remotexact/rwset.h (hunk 3 above)
Comment (poojanilangekar) [reply contrib/remotexact/rwset.h]: Done.

**Pull request merged**